app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,6 @@
  *= require_tree .
  *= require_self
  */
+ h1 {
+  text-align: center;
+}
```

Comment:
There should be end of line in each file, check VSCode settings, you can enable it on save.